ietf-document/draft-quic-4-sat/draft-kuhn-quic-4-sat-04.txt

```diff
@@ -0,0 +1,728 @@
+
+   o  85 s for downloading 100 MB
+
+   o  10 s for uploading 2 MB
+
+   o  50 s for uploading 10 MB
+
+   o  420 s for uploading 100 MB
+
+4.2.  Medium public satellite broadband access
+
+   The characteristics of the architecture under test are the following:
+
+   o  Satellite downlink path: 50 Mbps
+
+   o  Satellite uplink path: 10 Mbps
+
+   o  No emulated packet loss
+
+   o  RTT: 650 ms
+
+   o  Buffer size : BDP
+
+
+
+
+
+Kuhn, et al.             Expires August 31, 2020                [Page 6]
+
+Internet-Draft                 QUIC 4 SAT                  February 2020
+
+
+   During the transmission of 100 MB on the download path, the test
+   should report the downloading time of 2 MB, 10 MB and 100 MB.
+
+   Initial thoughts of the performance obectives for QUIC are the
+   following:
+
+   o  3 s for downloading 2 MB
+
+   o  7 s for downloading 10 MB
+
+   o  22 s for downloading 100 MB
+
+4.3.  Loss-free large public satellite broadband access
+
+   The characteristics of the architecture under test are the following:
+
+   o  Satellite downlink path: 250 Mbps
+
+   o  Satellite uplink path: 3 Mbps
+
+   o  No emulated packet loss
+
+   o  RTT: 650 ms
+
+   o  Buffer size : BDP
+
+   During the transmission of 100 MB on the download path, the test
+   should report the downloading time of 2 MB, 10 MB and 100 MB.
+
+   Initial thoughts of the performance obectives for QUIC are the
+   following:
+
+   o  3 s for downloading 2 MB
+
+   o  7 s for downloading 10 MB
+
+   o  10 s for downloading 100 MB
+
+4.4.  Lossy large public satellite broadband access
+
+   The characteristics of the architecture under test are the following:
+
+   o  Satellite downlink path: 250 Mbps
+
+   o  Satellite uplink path: 3 Mbps
+
+   o  Emulated packet loss on both downlink and uplink paths:
+
+
+
+
+Kuhn, et al.             Expires August 31, 2020                [Page 7]
+
+Internet-Draft                 QUIC 4 SAT                  February 2020
+
+
+      *  Uniform random transmission losses: 1%, or
+
+      *  Bursty losses, including transmission and congestion losses:
+         P(g|g)= 0.982, P(g|b)= 0.645, P(b|b)= 0.355, P(b|g)= 0.018 (see
+         [RFC6534] for more details)
+
+   o  RTT: 650 ms
+
+   o  Buffer size : BDP
+
+   During the transmission of 100 MB on the download path, the test
+   should report the downloading time of 2 MB, 10 MB and 100 MB.
+
+5.  Mechanisms that improve the performance of QUIC for SATCOM
+
+5.1.  Getting up to speed
+
+   The advantage of using QUIC is that it includes the TLS and TCP
+   negotiations that reduce the time at which the data can be
+   transmitted.  That being said, results of [IJSCN19] illustrate that
+   it will take many RTTs for the congestion controller to increase the
+   rate before it fills the bottleneck capacity.  This dominates
+   performance when a path has a large RTT (as in GEO SATCOM networks).
+   There is an issue in QUIC getting up to speed in a SATCOM context.
```

Comment:
/Is an issue/ or /Can be an issue/ ... it likely depends on the CC and other things also?